pkg/adapter/store.go

```diff
@@ -12,29 +12,29 @@ import (
 )
 
 const (
-	// KeyPrefix is the prefix used for all ABCI-related keys in the datastore
-	KeyPrefix = "abci"
+	// keyPrefix is the prefix used for all ABCI-related keys in the datastore
+	keyPrefix = "abci"
 	// stateKey is the key used for storing state
 	stateKey = "s"
 )
 
 // Store wraps a datastore with ABCI-specific functionality
 type Store struct {
-	ds.Batching
+	prefixedStore ds.Batching
```

Comment:
no need to have the abci store public at this stage when only load/save state is used